Networking/Networking/Requests/JetpackRequest.swift

```diff
@@ -138,6 +138,8 @@ private extension JetpackRequest {
             return nil
         }
 
-        return parameters.toJSONEncoded()
+        let jsonData = try? JSONSerialization.data(withJSONObject: parameters, options: [])
```

Comment:
Could this be encapsulated in an extension to Dictionary, similar to the way `toJSONEncoded()` works? (I am honestly not sure if it's even possible, but it would provide better encapsulation)

Reply:
Yes sure it's possible, but I've to include it inside a new Dictionary extension because `Any` does not conform to `Encodable`.
Done 👍